wolfssl/wolfcrypt/port/Renesas/renesas-tsip-crypt.h

```diff
@@ -87,16 +95,19 @@ enum {
 };
 
 typedef enum {
-    TSIP_KEY_TYPE_RSA2048      = 0,
-    TSIP_KEY_TYPE_RSA4096      = 1,
-    TSIP_KEY_TYPE_ECDSAP256    = 2,
     #ifdef WOLFSSL_RENESAS_TSIP_CRYPTONLY
-    TSIP_KEY_TYPE_RSA1024      = 3,
+    TSIP_KEY_TYPE_RSA1024      = 1, /* TSIP_RSA1024 */
```

Comment:
Can we not assign one to the other?

Reply:
The headers are a mess. I spent about an hour trying to get one to use the other (either direction) without success.